main.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+person = Person.new(22, 'maximilianus')
+puts person.correct_name # Output: maximilianus
+
+capitalized_person = CapitalizeDecorator.new(person)
+puts capitalized_person.correct_name # Output: Maximilianus
+
+capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+puts capitalized_trimmed_person.correct_name # Output: Maximilia
```

Comment:
- [ ] @jkanyi-web Great work adding the tests file, however a getting the error below when I run this file 
![image](https://github.com/jkanyi-web/OOP_School_Library/assets/61408860/69bfc083-72b5-4c99-8b84-69adc3f51c3c)